core/src/main/java/org/radargun/config/DomConfigParser.java

```diff
@@ -100,6 +102,20 @@ public MasterConfig parseConfig(String config) throws Exception {
       return masterConfig;
    }
 
+   private Element loadScenario(String scenarioFile) throws ParserConfigurationException {
```

Comment:
As for the documentation that @alanfx suggested, this is probably the piece of code that would deserve it most (as this is the main idea of the PR). I'd also change the name of parameter to 'scenarioUri'. The rest looks good to me. I'll integrate the changes once fixed.